docs/config/_default/params.toml

```diff
@@ -41,7 +41,7 @@ footer = ""
   breadCrumb = true
   highLight = true
   kaTex = false
-  multilingualMode = false
+  multilingualMode = true
```

Comment:
I don't think this should be enabled now. We don't have full Korean translation yet.

Reply:
Just the first commit is going to be merged, the second one was added so you can test the feature :D This is mentioned in the description of the PR :D